### ISSUE TITLE
Error parsing metadata on renamed package

### ISSUE THREAD

**Issue by Buckram123** [nextest-rs/nextest#1746]
nextest fails to parse Cargo metadata on renamed package
```
error: error parsing Cargo metadata

Caused by:
  failed to construct package graph: for package 'path+file:///...': no dependencies found matching 'big_package_name'
```

Minimal example:
```toml
[package]
name = "big-package-name"
version = "0.1.0"
edition = "2021"

[dependencies]

[dev-dependencies]
package = { path = ".", package = "big-package-name" }
```

Version used: v0.9.78

**Comment by sunshowers** [comment]
Thanks! An interesting case I hadn't seen before -- this is addressed in https://github.com/guppy-rs/guppy/pull/317 which I'll try getting out soon.

**Comment by sunshowers** [comment]
Just released a fix for this in cargo-nextest 0.9.79 -- binaries should be available within 15 minutes.

**Comment by Buckram123** [comment]
> Just released a fix for this in cargo-nextest 0.9.79 -- binaries should be available within 15 minutes.

Fixed, thank you!
